public/index.html

```diff
@@ -4,27 +4,27 @@
     <meta charset="utf-8" />
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
     <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
-    
+
     <link rel="preconnect" href="https://fonts.googleapis.com">
-    <link rel="preconnect" 
-          href="https://fonts.gstatic.com" 
+    <link rel="preconnect"
+          href="https://fonts.gstatic.com"
           crossorigin />
 
     <link rel="preload"
           as="style"
-          href="https://fonts.googleapis.com/css2?family=Oswald:wght@200&family=Anton&family=Codystar:wght@300&display=swap" />
+          href="https://fonts.googleapis.com/css2?family=Oswald:wght@200&family=Codystar:wght@300&display=block" />
 
     <link rel="stylesheet"
-          href="https://fonts.googleapis.com/css2?family=Oswald:wght@200&family=Anton&family=Codystar:wght@300&display=swap" 
+          href="https://fonts.googleapis.com/css2?family=Oswald:wght@200&family=Codystar:wght@300&display=block"
```

Comment:
This hide the text for max 3 seconds while the custom font-files are loaded.

Note that we’re not actually loading (or using) any custom font-file for the logo anymore, so you don’t necessarily need to set `display` to `block` here.

* `block` = Hide text for max 3secs (depending on browser) while font loads. If font-file is not done loading after 3secs then display anyway (in fallback/system font)
* `block` = Hide text for shorter time and do pretty much the same